REQUIRE

```diff
@@ -1,4 +1,4 @@
 julia 0.6
 URIParser
 SHA
-Compat 0.62.0
+Compat 0.65.1
```

Comment:
this can be left where it was, the 0.65.1 change was readme-only https://github.com/JuliaLang/Compat.jl/commit/2a822219a157b7fe0f45c6549fe4020c342ac29f

Reply:
done